src/common/core/coretype.js

```diff
@@ -398,21 +403,22 @@ define([
             }, TASYNC.lift(children));
         };
 
+        function checkCollNames(node, draft) {
```

Comment:
helper function should be positioned on the top of the file (driven by the editor-fold tags)